tls/extensions/s2n_extension_type.h

```diff
@@ -30,6 +30,7 @@
  * The +1 is necessary to handle any remainder left over when dividing. */
 #define S2N_SUPPORTED_EXTENSIONS_BITFIELD_LEN   ((S2N_SUPPORTED_EXTENSIONS_COUNT / sizeof(char)) + 1)
 
+struct s2n_connection;
```

Comment:
Why do we have this declaration? I don't see it being used anywhere within s2n_extension_type.c yet.

Reply:
It's an argument to multiple functions here. The compiler complains if we don't declare it.